rust/utils/run-locally/src/main.rs

```diff
@@ -476,38 +476,38 @@ fn fetch_metric(port: &str, metric: &str, labels: &HashMap<&str, &str>) -> Vec<u
                 .all(|(k, v)| l.contains(&format!("{}=\"{}\"", k, v)))
         })
         .map(|l| l.rsplit_once(' ').unwrap().1.parse::<u32>().unwrap())
-        .collect()
+        .collect())
```

Comment:
I think we should change this to something more like:

```rs
resp
    .call()?
    .into_string()?
    .lines()
    .filter(|l| l.starts_with(metric))
    .filter(|l| labels.iter().all(|(k, v)| l.contains(&format!("{k}=\"{v}\""))))
    .map(|l| Ok(
        l.rsplit_once(' ')
            .ok_or(eyre!("Unknown metric format"))?.1
            .parse::<u32>()?
    ))
    .collect()
```

This will catch the errors in parsing and `collect` should auto convert `Iterator<Item=Result<u32>>` into `Result<Vec<u32>>`